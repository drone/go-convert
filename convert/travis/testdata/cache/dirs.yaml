# https://docs.travis-ci.com/user/caching/#cache-rvm-ruby-version-for-non-ruby-projects

language: ruby

cache:
  directories:
  - /home/travis/.rvm/
