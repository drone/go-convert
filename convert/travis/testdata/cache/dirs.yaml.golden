kind: pipeline
spec:
  stages:
  - desc: converted from travis.yml
    name: pipeline
    spec:
      cache:
        enabled: true
        paths:
        - /home/travis/.rvm/
      steps:
      - name: install
        spec:
          run: bundle install --jobs=3 --retry=3
        type: script
      - name: script
        spec:
          run: rake
        type: script
    type: ci
version: 1
